.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
+.idea/
+
```

Comment:
## great start! we have a .gitignore file! but....
as a rule of thumb, we use `.gitignore` file to tell git what are the files we **don't** want to commit or push to repo. Your `.gitignore` file is a good start, but you can see there are files under `.gradle/` folder and under `build/` folder committed and pushed to github. 
Usually we don't need files in these 2 folders because:
- `.gradle/` folder stores metadata that `gradle` (the build tool) needs to get work done efficiently, like some lock files, history files.
- `build/` folder stores the compiled `.java` file, executable `jar` file, as well as test reports etc.

Other collaborators don't these files to build and run your Reminder app, that's why we don't commit these files.

---

## How to remove these files from your commit? 
Please take a look at this file: https://github.com/github/gitignore/blob/main/Gradle.gitignore
It is a very commonly used sample `.gitignore` file in the codebase. I'd suggest to update your file with this sample file.